src/main/app/authorized-axios/AuthorizedAxios.ts

```diff
@@ -104,9 +104,9 @@ export class AuthorizedAxios extends Axios {
 
   private requestInterceptor = () => {
```

Comment:
Is there a test that we can add for the interceptor?

Reply:
To be addressed in a future PR.

Reply:
As it's private and wouldn't make sense making public (especially just for unit testing) I will instead add a number of unit tests that just kind of test the AuthorizedAxios class as a whole. I'll do these later today (first within hmcts-access) as it's probably better to get this fix in now due to it affecting dashboard in PROD